modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliConfigKeys.cs

```diff
@@ -25,7 +25,7 @@ public static class LegacyCliConfigKeys {
         /// <summary>
         /// Key for the publisher site.
         /// </summary>
-        public const string PublisherSite = "Site";
+        public const string PublisherSite = "site";
```

Comment:
All other keys here start with an uppercase character, so this one should also be like them.

Reply:
oh, that's from merging back the main. The change was done to align the  site id with the other modules where is capital is not used.

Reply:
BTW, wasn't `Site` removed ?

Reply:
no, it's still there because the module host needs it. It's also part of twin and discovery modules as hardcoded "site" starting with small letter everywhere ...